### ISSUE TITLE
Keyboard navigation in DG2 does not work with listen-to-grid

### ISSUE THREAD

**Issue by mgroeneweg** [mendix/web-widgets#1927]
We have a DG2 with a sidebar on the right that listens to the grid, grid has single select with row click
The user cannot use the keyboard to navigate between grid lines and adjust the selection.
This works when checkbox selection is used as the checkbox itself can receive focus. But we use checkbox selection for multi-select only.

Keyboard navigation should update the selected row when using arrow keys, preferably without pressing enter or space as that invokes the default action on the grid.

**Comment by gjulivan** [comment]
NOTE: our current action to select is shift + space

**Comment by mgroeneweg** [comment]
Thanks for the update. Please note that it can be a little tricky to get the up/down arrow keys to work, you actually have to click in the grid but sometimes the arrow keys work and sometimes not, perhaps depents on where you click. Note that there is no custom content, only attribute columns
